seed.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ "${1-}" == "" ]; then
+  #Setup various users, organizations, and datasets
+  /usr/lib/ckan/bin/paster --plugin=ckan user add admin password=admin email=fake@fake.com -c /etc/ckan/production.ini > /tmp/user_temp.txt
+  /usr/lib/ckan/bin/paster --plugin=ckan sysadmin add admin -c /etc/ckan/production.ini
+  api_key=$(grep -oP "apikey.: u.\K.+" /tmp/user_temp.txt | cut -d "'" -f1)
+
+else
+  api_key="$1"
```

Comment:
Maybe just add a comment to the top of the file that the api_key can be passed in as the first argument.